### PR TITLE
refactor: migrate json-rpc to status-im2.common.json-rpc.events

### LINKED ISSUE #14597: move ::json-rpc/call proprly
we've duplicated ::json-rpc/call to status-im2 , but instead we should remove namespaced keyword and use 

`:json-rpc/call` in status-im2

remove `::json-rpc/call` from status-im 

replace ::json-rpc/call usage by `:json-rpc/call`

### PR DESCRIPTION
**Pull request by yqrashawn**
resolve #14597

1. remove status-im.ethereum.json-rpc
1. `::json-rpc/call` to `:json-rpc/call`
1. require `status-im2.common.json-rpc.events` in `status-im2.setup.events`
1. only new `:json-rpc/call` in mergeable keys

status: ready

### CODE REVIEW

File: src/status_im2/common/json_rpc/events.cljs [hunk 1]
```diff
@@ -45,7 +45,7 @@
                            (transforms/js->clj (.-result response-js)))))))))))
 
 (re-frame/reg-fx
- ::call
+ :call
```
Comment (flexsurfer) [inline src/status_im2/common/json_rpc/events.cljs]: :json-rpc/call

**Review by flexsurfer — APPROVED** [review]

File: src/status_im2/common/json_rpc/events.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im2/common/json_rpc/events.cljs]: @yqrashawn this is important, pls fix this

Comment (yqrashawn) [reply src/status_im2/common/json_rpc/events.cljs]: oops

Comment (yqrashawn) [reply src/status_im2/common/json_rpc/events.cljs]: fixed..

**Pull request merged**